editor/src/clj/editor/editor_extensions/ui_docs.clj

```diff
@@ -58,24 +87,20 @@
   {:alignment [:top-left :top :top-right :left :center :right :bottom-left :bottom :bottom-right]
    :padding [:small :medium :large]
    :spacing [:small :medium :large]
```

Comment:
I noticed the default value for `:spacing` varies between `editor.ui.horizontal` and `editor.ui.grid`. I think we should consistently use `editor.ui.SPACING.MEDIUM` for both, but we probably need to add `editor.ui.SPACING.NONE` and `editor.ui.PADDING.NONE` so you can explicitly disable spacing and padding?

Reply:
Huh, I don't see that the defaults are different.
Horizontal and Vertical default to medium here:
https://github.com/defold/defold/blob/75b5a5020abe6f288b5a9be6f1c82bad2851d47e/editor/src/clj/editor/editor_extensions/ui_components.clj#L277-L279
Grid defaults to medium here:
https://github.com/defold/defold/blob/75b5a5020abe6f288b5a9be6f1c82bad2851d47e/editor/src/clj/editor/editor_extensions/ui_components.clj#L240-L241

Reply:
Hmm, not sure where I got the idea that they were different to be honest. I must have caused it myself somehow when playing around with it. 😅